Renaming all AES_ Files and classes

nmahendru opened the pull request:
#### ba0f6471e7d465674ee660b9f4a6c34650639c7a
<pre>
Renaming all AES_ Files and classes
<a href="https://bugs.webkit.org/show_bug.cgi?id=268096">https://bugs.webkit.org/show_bug.cgi?id=268096</a>
<a href="https://rdar.apple.com/121541129">rdar://121541129</a>

Reviewed by David Kilzer.

This is a follow up change to this PR:
<a href="https://github.com/WebKit/WebKit/pull/22731">https://github.com/WebKit/WebKit/pull/22731</a>
The style checker complained about the underscores in the file
names that were changed in the mentioned PR.
We plan to change the other AES variants and thus this PR is
a standalone patch to make the style checker happy.

* Source/WebCore/Modules/webauthn/fido/Pin.cpp:
(fido::pin::TokenResponse::parse):
(fido::pin::encodeAsCBOR):
* Source/WebCore/Sources.txt:
* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCBC.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CBC.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESCBC):
(WebCore::CryptoAlgorithmAESCBC::create):
(WebCore::CryptoAlgorithmAESCBC::identifier const):
(WebCore::CryptoAlgorithmAESCBC::encrypt):
(WebCore::CryptoAlgorithmAESCBC::decrypt):
(WebCore::CryptoAlgorithmAESCBC::generateKey):
(WebCore::CryptoAlgorithmAESCBC::importKey):
(WebCore::CryptoAlgorithmAESCBC::exportKey):
(WebCore::CryptoAlgorithmAESCBC::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCBC.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CBC.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCFB.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CFB.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESCFB):
(WebCore::CryptoAlgorithmAESCFB::create):
(WebCore::CryptoAlgorithmAESCFB::identifier const):
(WebCore::CryptoAlgorithmAESCFB::encrypt):
(WebCore::CryptoAlgorithmAESCFB::decrypt):
(WebCore::CryptoAlgorithmAESCFB::generateKey):
(WebCore::CryptoAlgorithmAESCFB::importKey):
(WebCore::CryptoAlgorithmAESCFB::exportKey):
(WebCore::CryptoAlgorithmAESCFB::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCFB.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CFB.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCTR.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CTR.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESCTR):
(WebCore::parametersAreValid):
(WebCore::CryptoAlgorithmAESCTR::create):
(WebCore::CryptoAlgorithmAESCTR::identifier const):
(WebCore::CryptoAlgorithmAESCTR::encrypt):
(WebCore::CryptoAlgorithmAESCTR::decrypt):
(WebCore::CryptoAlgorithmAESCTR::generateKey):
(WebCore::CryptoAlgorithmAESCTR::importKey):
(WebCore::CryptoAlgorithmAESCTR::exportKey):
(WebCore::CryptoAlgorithmAESCTR::getKeyLength):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockHelper):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::countToOverflowSaturating const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::counterVectorAfterOverflow const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::set):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::all const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::any const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator&amp; const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator~ const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator &lt;&lt;=):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator &amp;=):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCTR.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CTR.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESGCM.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_GCM.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESGCM):
(WebCore::tagLengthIsValid):
(WebCore::CryptoAlgorithmAESGCM::create):
(WebCore::CryptoAlgorithmAESGCM::identifier const):
(WebCore::CryptoAlgorithmAESGCM::encrypt):
(WebCore::CryptoAlgorithmAESGCM::decrypt):
(WebCore::CryptoAlgorithmAESGCM::generateKey):
(WebCore::CryptoAlgorithmAESGCM::importKey):
(WebCore::CryptoAlgorithmAESGCM::exportKey):
(WebCore::CryptoAlgorithmAESGCM::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESGCM.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_GCM.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESKW.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_KW.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESKW):
(WebCore::CryptoAlgorithmAESKW::create):
(WebCore::CryptoAlgorithmAESKW::identifier const):
(WebCore::CryptoAlgorithmAESKW::generateKey):
(WebCore::CryptoAlgorithmAESKW::importKey):
(WebCore::CryptoAlgorithmAESKW::exportKey):
(WebCore::CryptoAlgorithmAESKW::wrapKey):
(WebCore::CryptoAlgorithmAESKW::unwrapKey):
(WebCore::CryptoAlgorithmAESKW::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESKW.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_KW.h.
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESCBCGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_CBCGCrypt.cpp.
(WebCore::gcryptEncrypt):
(WebCore::gcryptDecrypt):
(WebCore::CryptoAlgorithmAESCBC::platformEncrypt):
(WebCore::CryptoAlgorithmAESCBC::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESCFBGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_CFBGCrypt.cpp.
(WebCore::CryptoAlgorithmAESCFB::platformEncrypt):
(WebCore::CryptoAlgorithmAESCFB::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESCTRGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_CTRGCrypt.cpp.
(WebCore::callOperation):
(WebCore::gcryptAES_CTR):
(WebCore::CryptoAlgorithmAESCTR::platformEncrypt):
(WebCore::CryptoAlgorithmAESCTR::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESGCMGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_GCMGCrypt.cpp.
(WebCore::gcryptEncrypt):
(WebCore::gcryptDecrypt):
(WebCore::CryptoAlgorithmAESGCM::platformEncrypt):
(WebCore::CryptoAlgorithmAESGCM::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESKWGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_KWGCrypt.cpp.
(WebCore::gcryptWrapKey):
(WebCore::gcryptUnwrapKey):
(WebCore::CryptoAlgorithmAESKW::platformWrapKey):
(WebCore::CryptoAlgorithmAESKW::platformUnwrapKey):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmRegistryGCrypt.cpp:
(WebCore::CryptoAlgorithmRegistry::platformRegisterAlgorithms):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESCBCMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_CBCMac.cpp.
(WebCore::transformAES_CBC):
(WebCore::CryptoAlgorithmAESCBC::platformEncrypt):
(WebCore::CryptoAlgorithmAESCBC::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESCFBMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_CFBMac.cpp.
(WebCore::transformAES_CFB):
(WebCore::CryptoAlgorithmAESCFB::platformEncrypt):
(WebCore::CryptoAlgorithmAESCFB::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESCTRMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_CTRMac.cpp.
(WebCore::CryptoAlgorithmAESCTR::platformEncrypt):
(WebCore::CryptoAlgorithmAESCTR::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESGCMMac.mm: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_GCMMac.mm.
(WebCore::encryptAES_GCM):
(WebCore::encryptCryptoKitAES_GCM):
(WebCore::decyptAES_GCM):
(WebCore::CryptoAlgorithmAESGCM::platformEncrypt):
(WebCore::CryptoAlgorithmAESGCM::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESKWMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_KWMac.cpp.
(WebCore::wrapKeyAES_KW):
(WebCore::unwrapKeyAES_KW):
(WebCore::CryptoAlgorithmAESKW::platformWrapKey):
(WebCore::CryptoAlgorithmAESKW::platformUnwrapKey):
* Source/WebCore/crypto/mac/CryptoAlgorithmRegistryMac.cpp:
(WebCore::CryptoAlgorithmRegistry::platformRegisterAlgorithms):
* Source/WebCore/crypto/mac/CryptoUtilitiesCocoa.cpp:
(WebCore::transformAES_CTR):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESCBCOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_CBCOpenSSL.cpp.
(WebCore::aesAlgorithm):
(WebCore::cryptEncrypt):
(WebCore::cryptDecrypt):
(WebCore::CryptoAlgorithmAESCBC::platformEncrypt):
(WebCore::CryptoAlgorithmAESCBC::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESCFBOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_CFBOpenSSL.cpp.
(WebCore::cfb8):
(WebCore::cryptEncrypt):
(WebCore::cryptDecrypt):
(WebCore::CryptoAlgorithmAESCFB::platformEncrypt):
(WebCore::CryptoAlgorithmAESCFB::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESCTROpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_CTROpenSSL.cpp.
(WebCore::aesAlgorithm):
(WebCore::crypt):
(WebCore::CryptoAlgorithmAESCTR::platformEncrypt):
(WebCore::CryptoAlgorithmAESCTR::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESGCMOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_GCMOpenSSL.cpp.
(WebCore::aesAlgorithm):
(WebCore::cryptEncrypt):
(WebCore::cryptDecrypt):
(WebCore::CryptoAlgorithmAESGCM::platformEncrypt):
(WebCore::CryptoAlgorithmAESGCM::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESKWOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_KWOpenSSL.cpp.
(WebCore::cryptWrapKey):
(WebCore::cryptUnwrapKey):
(WebCore::CryptoAlgorithmAESKW::platformWrapKey):
(WebCore::CryptoAlgorithmAESKW::platformUnwrapKey):
* Source/WebCore/crypto/openssl/CryptoAlgorithmRegistryOpenSSL.cpp:
(WebCore::CryptoAlgorithmRegistry::platformRegisterAlgorithms):
* Source/WebCore/crypto/openssl/SerializedCryptoKeyWrapOpenSSL.cpp:
* Tools/TestWebKitAPI/Tests/WebCore/CtapPinTest.cpp:
(TestWebKitAPI::TEST):

Canonical link: <a href="https://commits.webkit.org/273566@main">https://commits.webkit.org/273566@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e2d3f90e8da87bc07c7f7f24026129020a0981fe

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/35794 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/14737 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/37940 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/38519 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/32211 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/37012 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/17171 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/11761 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/30967 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/36348 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/12475 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/31816 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/10940 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/10948 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/31987 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/39767 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/32539 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/32309 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/36884 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/11128 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/9014 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/34974 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/12868 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/11633 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/4643 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/11946 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

ddkilzer reviewed (approved):
rs=me  (rs = rubberstamp because there are too many individual changes to review)

nmahendru committed:
Renaming all AES_ Files and classes
https://bugs.webkit.org/show_bug.cgi?id=268096
rdar://121541129

Reviewed by David Kilzer.

This is a follow up change to this PR:
https://github.com/WebKit/WebKit/pull/22731
The style checker complained about the underscores in the file
names that were changed in the mentioned PR.
We plan to change the other AES variants and thus this PR is
a standalone patch to make the style checker happy.

* Source/WebCore/Modules/webauthn/fido/Pin.cpp:
(fido::pin::TokenResponse::parse):
(fido::pin::encodeAsCBOR):
* Source/WebCore/Sources.txt:
* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCBC.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CBC.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESCBC):
(WebCore::CryptoAlgorithmAESCBC::create):
(WebCore::CryptoAlgorithmAESCBC::identifier const):
(WebCore::CryptoAlgorithmAESCBC::encrypt):
(WebCore::CryptoAlgorithmAESCBC::decrypt):
(WebCore::CryptoAlgorithmAESCBC::generateKey):
(WebCore::CryptoAlgorithmAESCBC::importKey):
(WebCore::CryptoAlgorithmAESCBC::exportKey):
(WebCore::CryptoAlgorithmAESCBC::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCBC.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CBC.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCFB.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CFB.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESCFB):
(WebCore::CryptoAlgorithmAESCFB::create):
(WebCore::CryptoAlgorithmAESCFB::identifier const):
(WebCore::CryptoAlgorithmAESCFB::encrypt):
(WebCore::CryptoAlgorithmAESCFB::decrypt):
(WebCore::CryptoAlgorithmAESCFB::generateKey):
(WebCore::CryptoAlgorithmAESCFB::importKey):
(WebCore::CryptoAlgorithmAESCFB::exportKey):
(WebCore::CryptoAlgorithmAESCFB::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCFB.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CFB.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCTR.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CTR.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESCTR):
(WebCore::parametersAreValid):
(WebCore::CryptoAlgorithmAESCTR::create):
(WebCore::CryptoAlgorithmAESCTR::identifier const):
(WebCore::CryptoAlgorithmAESCTR::encrypt):
(WebCore::CryptoAlgorithmAESCTR::decrypt):
(WebCore::CryptoAlgorithmAESCTR::generateKey):
(WebCore::CryptoAlgorithmAESCTR::importKey):
(WebCore::CryptoAlgorithmAESCTR::exportKey):
(WebCore::CryptoAlgorithmAESCTR::getKeyLength):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockHelper):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::countToOverflowSaturating const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::counterVectorAfterOverflow const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::set):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::all const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::any const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator& const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator~ const):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator <<=):
(WebCore::CryptoAlgorithmAESCTR::CounterBlockHelper::CounterBlockBits::operator &=):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESCTR.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_CTR.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESGCM.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_GCM.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESGCM):
(WebCore::tagLengthIsValid):
(WebCore::CryptoAlgorithmAESGCM::create):
(WebCore::CryptoAlgorithmAESGCM::identifier const):
(WebCore::CryptoAlgorithmAESGCM::encrypt):
(WebCore::CryptoAlgorithmAESGCM::decrypt):
(WebCore::CryptoAlgorithmAESGCM::generateKey):
(WebCore::CryptoAlgorithmAESGCM::importKey):
(WebCore::CryptoAlgorithmAESGCM::exportKey):
(WebCore::CryptoAlgorithmAESGCM::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESGCM.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_GCM.h.
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESKW.cpp: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_KW.cpp.
(WebCore::usagesAreInvalidForCryptoAlgorithmAESKW):
(WebCore::CryptoAlgorithmAESKW::create):
(WebCore::CryptoAlgorithmAESKW::identifier const):
(WebCore::CryptoAlgorithmAESKW::generateKey):
(WebCore::CryptoAlgorithmAESKW::importKey):
(WebCore::CryptoAlgorithmAESKW::exportKey):
(WebCore::CryptoAlgorithmAESKW::wrapKey):
(WebCore::CryptoAlgorithmAESKW::unwrapKey):
(WebCore::CryptoAlgorithmAESKW::getKeyLength):
* Source/WebCore/crypto/algorithms/CryptoAlgorithmAESKW.h: Renamed from Source/WebCore/crypto/algorithms/CryptoAlgorithmAES_KW.h.
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESCBCGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_CBCGCrypt.cpp.
(WebCore::gcryptEncrypt):
(WebCore::gcryptDecrypt):
(WebCore::CryptoAlgorithmAESCBC::platformEncrypt):
(WebCore::CryptoAlgorithmAESCBC::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESCFBGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_CFBGCrypt.cpp.
(WebCore::CryptoAlgorithmAESCFB::platformEncrypt):
(WebCore::CryptoAlgorithmAESCFB::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESCTRGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_CTRGCrypt.cpp.
(WebCore::callOperation):
(WebCore::gcryptAES_CTR):
(WebCore::CryptoAlgorithmAESCTR::platformEncrypt):
(WebCore::CryptoAlgorithmAESCTR::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESGCMGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_GCMGCrypt.cpp.
(WebCore::gcryptEncrypt):
(WebCore::gcryptDecrypt):
(WebCore::CryptoAlgorithmAESGCM::platformEncrypt):
(WebCore::CryptoAlgorithmAESGCM::platformDecrypt):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmAESKWGCrypt.cpp: Renamed from Source/WebCore/crypto/gcrypt/CryptoAlgorithmAES_KWGCrypt.cpp.
(WebCore::gcryptWrapKey):
(WebCore::gcryptUnwrapKey):
(WebCore::CryptoAlgorithmAESKW::platformWrapKey):
(WebCore::CryptoAlgorithmAESKW::platformUnwrapKey):
* Source/WebCore/crypto/gcrypt/CryptoAlgorithmRegistryGCrypt.cpp:
(WebCore::CryptoAlgorithmRegistry::platformRegisterAlgorithms):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESCBCMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_CBCMac.cpp.
(WebCore::transformAES_CBC):
(WebCore::CryptoAlgorithmAESCBC::platformEncrypt):
(WebCore::CryptoAlgorithmAESCBC::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESCFBMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_CFBMac.cpp.
(WebCore::transformAES_CFB):
(WebCore::CryptoAlgorithmAESCFB::platformEncrypt):
(WebCore::CryptoAlgorithmAESCFB::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESCTRMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_CTRMac.cpp.
(WebCore::CryptoAlgorithmAESCTR::platformEncrypt):
(WebCore::CryptoAlgorithmAESCTR::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESGCMMac.mm: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_GCMMac.mm.
(WebCore::encryptAES_GCM):
(WebCore::encryptCryptoKitAES_GCM):
(WebCore::decyptAES_GCM):
(WebCore::CryptoAlgorithmAESGCM::platformEncrypt):
(WebCore::CryptoAlgorithmAESGCM::platformDecrypt):
* Source/WebCore/crypto/mac/CryptoAlgorithmAESKWMac.cpp: Renamed from Source/WebCore/crypto/mac/CryptoAlgorithmAES_KWMac.cpp.
(WebCore::wrapKeyAES_KW):
(WebCore::unwrapKeyAES_KW):
(WebCore::CryptoAlgorithmAESKW::platformWrapKey):
(WebCore::CryptoAlgorithmAESKW::platformUnwrapKey):
* Source/WebCore/crypto/mac/CryptoAlgorithmRegistryMac.cpp:
(WebCore::CryptoAlgorithmRegistry::platformRegisterAlgorithms):
* Source/WebCore/crypto/mac/CryptoUtilitiesCocoa.cpp:
(WebCore::transformAES_CTR):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESCBCOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_CBCOpenSSL.cpp.
(WebCore::aesAlgorithm):
(WebCore::cryptEncrypt):
(WebCore::cryptDecrypt):
(WebCore::CryptoAlgorithmAESCBC::platformEncrypt):
(WebCore::CryptoAlgorithmAESCBC::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESCFBOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_CFBOpenSSL.cpp.
(WebCore::cfb8):
(WebCore::cryptEncrypt):
(WebCore::cryptDecrypt):
(WebCore::CryptoAlgorithmAESCFB::platformEncrypt):
(WebCore::CryptoAlgorithmAESCFB::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESCTROpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_CTROpenSSL.cpp.
(WebCore::aesAlgorithm):
(WebCore::crypt):
(WebCore::CryptoAlgorithmAESCTR::platformEncrypt):
(WebCore::CryptoAlgorithmAESCTR::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESGCMOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_GCMOpenSSL.cpp.
(WebCore::aesAlgorithm):
(WebCore::cryptEncrypt):
(WebCore::cryptDecrypt):
(WebCore::CryptoAlgorithmAESGCM::platformEncrypt):
(WebCore::CryptoAlgorithmAESGCM::platformDecrypt):
* Source/WebCore/crypto/openssl/CryptoAlgorithmAESKWOpenSSL.cpp: Renamed from Source/WebCore/crypto/openssl/CryptoAlgorithmAES_KWOpenSSL.cpp.
(WebCore::cryptWrapKey):
(WebCore::cryptUnwrapKey):
(WebCore::CryptoAlgorithmAESKW::platformWrapKey):
(WebCore::CryptoAlgorithmAESKW::platformUnwrapKey):
* Source/WebCore/crypto/openssl/CryptoAlgorithmRegistryOpenSSL.cpp:
(WebCore::CryptoAlgorithmRegistry::platformRegisterAlgorithms):
* Source/WebCore/crypto/openssl/SerializedCryptoKeyWrapOpenSSL.cpp:
* Tools/TestWebKitAPI/Tests/WebCore/CtapPinTest.cpp:
(TestWebKitAPI::TEST):

Canonical link: https://commits.webkit.org/273566@main
